samples/sets/add.jakt

```diff
@@ -0,0 +1,9 @@
+function main() {
+    let mutable set = {"b", "c"}
+    set.add("a")
+    println("{}", set.size())
+
+    set.add("b")
+    set.add("c")
+    println("{}", set.size())
+}
```

Comment:
Don't know if this is considered an issue, but these files are ending missing newlines